appengine/standard_python3/bundled-services/deferred/django/main.py

```diff
@@ -28,8 +29,10 @@
 
 class Counter(ndb.Model):
     count = ndb.IntegerProperty(indexed=False)
+    updated_at = ndb.DateTimeProperty(auto_now=True)
```

Comment:
Just curious, how does having the timestamp show will remove them from the database? Or is that planned as TODO?

Reply:
If there's a timestamp, I can have a policy remove them after a certain time. But I'm not going to move forward with this PR.